### PR TITLE
build: debian: small series of lintian errors addressed

### PR DESCRIPTION
**Pull request by smoe**
We have quite a few complaints by the Debian bots that check on the package's compliance with Debian's packaging policy on https://udd.debian.org/lintian/?packages=linuxcnc .
This PR addressed a small (admittedly) number of them.

The warning on the man page was
W: linuxcnc-uspace: groff-message troff:<standard input>:209: warning: macro 'an-trap' not defined [usr/share/man/man1/xhc-whb04b-6.1.gz:1] which was not immediately obvious and triggered a quick complete review of the corresponding .adoc.

### CODE REVIEW

**Comment by smoe** [comment]
Rebased on master to get CI complete.

To add to the initial summary: The d/copyright has files now sorted alphabetically, so this helps avoiding future redundancies.  The troff warning was likely a problem of asciidoc->man conversion, I just tried to simplify things.

File: docs/src/man/man1/xhc-whb04b-6.1.adoc [hunk 1]
```diff
@@ -88,22 +76,23 @@ should be created with the single line +
...
 `xhc-whb04b-6 -p`::
-Start in simulation mode and prints HAL pin names and events distributed to HAL memory.
+ Start in simulation mode and prints HAL pin names and events distributed to HAL memory.
 `xhc-whb04b-6 -H`::
-Start in HAL mode (Normal mode for real machine use).
+ Start in HAL mode (Normal mode for real machine use).
 `xhc-whb04b-6 -HsfB`::
-Start in HAL mode + Spindle Override + Feedrate Override + Big step 5/10&#8239;mm).
-
+ Start in HAL mode + Spindle Override + Feedrate Override + Big step 5/10&#8239;mm).
```
Comment (petterreinholdtsen) [inline docs/src/man/man1/xhc-whb04b-6.1.adoc:92]: Will the removal of ' +' at the end of the line and before 'Big step', remove a newline?  Is there a '(' missing here?  I only see the ')'.

File: lib/python/qtvcp/lib/mdi_text.py
Comment (petterreinholdtsen) [inline lib/python/qtvcp/lib/mdi_text.py:1]: Why is this empty file needed?

**Comment by petterreinholdtsen** [comment]
Most changes look great to me, but I have a few which made me wonder, see individual comments.

File: lib/python/qtvcp/lib/mdi_text.py
Comment (smoe) [reply lib/python/qtvcp/lib/mdi_text.py:1]: Permission change that Lintian requested -  it is executable now. Have not checked if the file has any content.

Comment (petterreinholdtsen) [reply lib/python/qtvcp/lib/mdi_text.py:1]: Ah, so it is only empty in the diff, not on disk.  My mistake.

File: docs/src/man/man1/xhc-whb04b-6.1.adoc (hunk 1 above)
Comment (smoe) [reply docs/src/man/man1/xhc-whb04b-6.1.adoc]: Added the missing '(' , everything else looks as it should look (IMHO), except that you would say "0.5 mm" or "1/2 mm", but not "5/10 mm".

File: lib/python/qtvcp/lib/mdi_text.py
Comment (smoe) [reply lib/python/qtvcp/lib/mdi_text.py:1]: Checked now, the file is not empty.

**Comment by petterreinholdtsen** [comment]
[Steffen Möller]
>>  `xhc-whb04b-6 -HsfB`::
> -Start in HAL mode + Spindle Override + Feedrate Override + Big step 5/10&#8239;mm).
> -
> + Start in HAL mode + Spindle Override + Feedrate Override + Big step 5/10&#8239;mm).
>
> Added the missing '(' , everything else looks as it should look
> (IMHO), except that you would say "0.5 mm" or "1/2 mm", but not "5/10
> mm".

Are you sure the big step is not 5-10 mm, not 0.5mm?

-- 
Happy hacking
Petter Reinholdtsen

**Comment by smoe** [comment]
> [Steffen Möller]
> > `xhc-whb04b-6 -HsfB`:: -Start in HAL mode + Spindle Override + Feedrate Override + Big step 5/10&#8239;mm). - + Start in HAL mode + Spindle Override + Feedrate Override + Big step 5/10&#8239;mm). Added the missing '(' , everything else looks as it should look (IMHO), except that you would say "0.5 mm" or "1/2 mm", but not "5/10 mm".
> Are you sure the big step is not 5-10 mm, not 0.5mm?
> […](#)
> -- Happy hacking Petter Reinholdtsen

No, I am not sure, thought it was meant to help those who are less familiar with the metric system. But 5-10 mm sound more like giant steps than mere big steps. My patch was meant to be purely technical and I am confident that it improves over the current state.

**Pull request merged**